ext/node/ops/crypto/mod.rs

```diff
@@ -362,18 +362,33 @@ pub fn op_node_sign(
   #[cppgc] handle: &KeyObjectHandle,
   #[buffer] digest: &[u8],
   #[string] digest_type: &str,
+  #[smi] pss_salt_length: Option<u32>,
+  #[smi] dsa_signature_encoding: u32,
 ) -> Result<Box<[u8]>, AnyError> {
-  handle.sign_prehashed(digest_type, digest)
+  handle.sign_prehashed(
+    digest_type,
+    digest,
+    pss_salt_length,
+    dsa_signature_encoding,
+  )
 }
 
-#[op2(fast)]
+#[op2]
 pub fn op_node_verify(
   #[cppgc] handle: &KeyObjectHandle,
   #[buffer] digest: &[u8],
   #[string] digest_type: &str,
   #[buffer] signature: &[u8],
+  #[smi] pss_salt_length: Option<u32>,
```

Comment:
We could use one param to represent both fields. They are mutually exclusive.

May be marginally faster, but probably not worth it.

Reply:
It will be worth it if we could get rid of `Option` and make this fast call. Will skip for now.